spec/factories.rb

```diff
@@ -24,6 +24,11 @@
     published_at { Time.zone.now }
   end
 
+  factory :published_step_by_step_page_live, parent: :step_by_step_page_with_steps do
```

Comment:
Can `published_step_by_step_page` reused instead of creating a new factory?

Reply:
No, as the draft date is separate to the published date, so `step_nav.status` is actually `unpublished_changes` instead of `live`.

(I think it's badly named and should be renamed to `published_step_by_step_with_changes`, and my own factory should be `published_step_by_step`!)

Reply:
Yes, I think `published_step_by_step_page` should be changed to return a step by step that has a status of "live".

If there are any tests that fail because they rely on the status being "unpublished changes", then a new factory could be added. 

If you need to create a new factory I'd recommend doing these changes in a new commit. Otherwise, if you are only changing the `draft_updated_at` time in `published_step_by_step_page` then a line in the commit note explaining why should be enough.

Reply:
That's now done 👍 one of the tests did fail, so I've encapsulated it in a commit before the main commit of the PR. Putting it as the second commit would've required making a change to something I did in the first commit.